### PR TITLE
Use Slack and Influx handler assets

### LINKED ISSUE #1240: Update docs to use Slack and InfluxDB assets
Update the Slack and InfluxDB guides to use assets.

https://bonsai.sensu.io/assets/sensu/sensu-influxdb-handler
https://bonsai.sensu.io/assets/sensu/sensu-slack-handler

- [ ] https://docs.sensu.io/sensu-go/5.2/guides/send-slack-alerts/
- [ ] https://docs.sensu.io/sensu-go/5.2/guides/influx-db-metric-handler/
- [ ] Update Bonsai copy to reference these assets
- [ ] Learn Sensu Go sandbox lesson

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Updates the Slack guide, Influx guide, and sandbox intro lesson to use the Slack and Influx handler assets

## Motivation and Context

Closes #1240

### CODE REVIEW

**Comment by apaskulin** [comment]
This PR requires a follow up to apply changes to version 5.3.

**Review by rgeniesse — DISMISSED** [review]
LGTM. Ran through the commands and they add assets as intended.

**Review by rgeniesse — COMMENTED** [review]
While I think it is great we are showing both ways to create resources using `sensuctl`, the sandbox already has definitions for the handler and assets. So if we do use `sensuctl create` to do some of the work, we'll want to get rid of the corresponding JSON files. Or just not use `sensuctl create` and rely on the sandbox to continue to provide those definitions.

An argument could be made that if this is mostly aimed at new users of Sensu Go, that using both could lead to confusion.

File: content/sensu-go/5.2/getting-started/learn-sensu.md [hunk 1]
```diff
@@ -301,24 +335,46 @@ Now let's create the InfluxDB handler to store these metrics and visualize them
...
+You can use sensuctl to see the complete asset definition.
+
+{{< highlight shell >}}
+sensuctl asset info sensu-influxdb-handler --format yaml
+{{< /highlight >}}
+
+Open the `influx-handler.json` handler definition provided with the sandbox, and edit the `runtime_assets` attribute to include the `sensu-influxdb-handler` asset.
+
+{{< highlight shell >}}
+"runtime_assets": ["sensu-slack-handler"]
```
Comment (rgeniesse) [inline content/sensu-go/5.2/getting-started/learn-sensu.md]:
```suggestion
"runtime_assets": ["sensu-influxdb-handler"]
```

File: content/sensu-go/5.2/getting-started/learn-sensu.md [hunk 2]
```diff
@@ -301,24 +335,46 @@ Now let's create the InfluxDB handler to store these metrics and visualize them
 
 **4. Create an InfluxDB pipeline**
 
-Since we've already installed InfluxDB as part of the sandbox, all we need to do to create an InfluxDB pipeline is create a Sensu handler.
+To create a pipeline to send metric events to InfluxDB, start by registering the [Sensu InfluxDB handler asset][4].
+
+{{< highlight shell >}}
+sensuctl asset create sensu-influxdb-handler --url "https://github.com/sensu/sensu-influxdb-handler/releases/download/3.1.2/sensu-influxdb-handler_3.1.2_linux_arm64.tar.gz" --sha512 "9936dce7416345b98df1076673b5713d0a9f1d7aebd02e4a3794e491d4716e039acae16a4360030b0bc3128f6fced3abc42e3208a55d8ff89e645cc5c3905765"
```
Comment (rgeniesse) [inline content/sensu-go/5.2/getting-started/learn-sensu.md]:
```suggestion
sensuctl asset create sensu-influxdb-handler --url "https://github.com/sensu/sensu-influxdb-handler/releases/download/3.1.2/sensu-influxdb-handler_3.1.2_linux_amd64.tar.gz" --sha512 "612c6ff9928841090c4d23bf20aaf7558e4eed8977a848cf9e2899bb13a13e7540bac2b63e324f39d9b1257bb479676bc155b24e21bf93c722b812b0f15cb3bd"
```

**Review by rgeniesse — APPROVED** [review]
💯

**Pull request merged**